browser/brave_stats/brave_stats_updater_unittest.cc

```diff
@@ -259,8 +259,8 @@ TEST_F(BraveStatsUpdaterTest, HasDailyRetention) {
   exploded.day_of_month = 4;
 
   EXPECT_TRUE(base::Time::FromLocalExploded(exploded, &dtoi_time));
-  // Make first run date 6 days earlier (still within 14 day window)
-  exploded.day_of_month = 10;
+  // Make first run date 15 days earlier (still within 14 day window)
```

Comment:
Nit: I think maybe we need to replace `14` with `30` in this comment...?

Reply:
thanks. fixed.